Library/Formula/srm.rb

```diff
@@ -0,0 +1,74 @@
+class Srm < Formula
+  homepage "http://www.thc.org/"
+  url "https://www.thc.org/releases/secure_delete-3.1.tar.gz"
+  sha256 "a9d846d1dce3f1bdf13bbb306e8596bc1f263198a086f6beecd90ccf7bddf8d5"
+
+  keg_only :provided_by_osx if MacOS.version >= :mavericks
+
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!